draft-ietf-httpbis-priority.md

```diff
@@ -412,16 +412,18 @@ might depend heavily on one of the inline images. Existence of such
 dependencies is typically best known to the server.
 
 An origin can use the Priority response header field to indicate its view on how
-an HTTP response should be prioritized. When forwarding an HTTP response with
-the Priority response header field, an intermediary can use the parameters being
-found in that response instead of those found in the request.
+an HTTP response should be prioritized. An intermediary that forwards an HTTP
+response can use the parameters found in the Priority response header field, in
+combination with the client Priority request header field, as input to its
+prioritization process. No guidance is provided for merging priorities, this is
+left as an implementation decision.
 
 Absence of a priority parameter in an HTTP response indicates the server's
 disinterest in changing the client-provided value. This is different from the
 logic being defined for the request header field, in which omission of a
 priority parameter implies the use of their default values (see #parameters).
 
-For example, when the client sends an HTTP request with
+A non-normative example, when the client sends an HTTP request with
```

Comment:
To a non-native speaker, "As a non-normative example" sounds more natural, though I am not sure if it's actually better.